docs/advanced-features/compiler.md

```diff
@@ -312,13 +319,36 @@ module.exports = {
 }
 ```
 
-Advanced transformations:
+#### Handlebars variables and helper functions
+
+This transform uses [handlebars](https://docs.rs/handlebars) to template the replacement import path in the `transform` field. These variables and helper functions are available:
+
+1. `member`: Has type `string`. The name of the member import.
+2. `lowerCase`, `upperCase`, `camelCase`, `kebabCase`: Helper functions to convert a string to lower, upper, camel or kebab cases.
+3. `matches`: Has type `string[]`. All groups matched by the regular expression. `matches.[0]` is the full match.
+
+For example, you can use the `kebabCase` helper like this:
+
+```js
+// next.config.js
+module.exports = {
+  'my-library': {
+    transform: 'my-library/{{ kebabCase member }}',
+  },
+}
+```
+
+The above config will transform your code as follows:
 
-- Using regular expressions
+```js
+// Before
+import { MyModule } from 'my-library'
 
-Similar to `babel-plugin-transform-imports`, but the transform is templated with [handlebars](https://docs.rs/handlebars) and regular expressions are in Rust [regex](https://docs.rs/regex/latest/regex/) crate's syntax.
+// After (`MyModule` was converted to `my-module`)
+import MyModule from 'my-library/my-module'
+```
```

Comment:
Added an example of how to use the helper functions (e.g. `kebabCase`) — previously, you had to look up the linked handlebars documentation to learn how to use helper functions.

Also, I used the `my-library`, `MyModule`, etc example which was already used in the regex example that follows.